internal/log/log.go

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
How about a directory structure like this?
```
cmd/ # only for the command line parsing, and a single call to the conversion package
pkg/
      conversion/ # The core conversion code
      k8sutil/    # kubernetes helpers
      log/        # logging helpers
```

Reply:
The directory structure is now updated as suggested :+1: